### PR TITLE
Fix back action in group detail

### PR DESCRIPTION
**Pull request by fhlavac**
### Description

Fixed back browser action in group detail

[RHCLOUD-29600](https://issues.redhat.com/browse/RHCLOUD-29600)

---

#### Before:
chrome-capture-2024-7-13 (2)

#### After:
chrome-capture-2024-7-13 (1)

---

### Checklist ☑️
- [x] PR only fixes one issue or story 
- [x] Change reviewed for extraneous code 
- [x] UI best practices adhered to 
- [x] Commits squashed and meaningfully named 
- [x] All PR checks pass locally (build, lint, test, E2E)

##
- [ ] _(Optional) QE: Needs QE attention (OUIA changed, perceived impact to tests, no test coverage)_
- [ ] _(Optional) QE: Has been mentioned_
- [ ] _(Optional) UX: Needs UX attention (end user UX modified, missing designs)_
- [ ] _(Optional) UX: Has been mentioned_
##

### CODE REVIEW

File: src/test/smart-components/group/service-account/__snapshots__/add-group-service-accounts.test.js.snap [hunk 1]
```diff
@@ -733,7 +733,7 @@ exports[`<AddGroupServiceAccounts /> should render correctly with data 1`] = `
                             style="display: contents;"
                           >
                             <span>
-                              54 years ago
+                              55 years ago
```
Comment (aneelac22) [inline src/test/smart-components/group/service-account/__snapshots__/add-group-service-accounts.test.js.snap:736]: Not sure why this change is needed?

Comment (Hyperkid123) [reply src/test/smart-components/group/service-account/__snapshots__/add-group-service-accounts.test.js.snap:736]: Seems like there is a specific date somewhere in the tests and we just got over the year threshold
